admin/c2cgeoportal_admin/templates/navigation_navbar.jinja2

```diff
@@ -28,8 +28,9 @@
         <li id="language-dropdown" class="dropdown">
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{request.locale_name}} <span class="caret"></span></a>
           <ul class="dropdown-menu">
-            <li><a href="{{request.route_url('locale', _query=(('language', 'en'),))}}">en</a></li>
-            <li><a href="{{request.route_url('locale', _query=(('language', 'fr'),))}}">fr</a></li>
+            {% for locale in request.registry.settings['available_locale_names']|sort() %}
```

Comment:
for me this syntax looks strange, and why do we sort it, for me we should keep the order...

Reply:
Concerning the syntax, it is a jinja2 file, here I use a filter. For me, it is common to sort entries in dropdown lists. This is just : user friendly.

Reply:
For me it's the integrator who should be able to do what he want...